tests/test_plotting.py

```diff
@@ -36,6 +36,17 @@ def test_plot_frequency_recency_matrix(self):
 
         plt.show()
 
+    def test_plot_probability_alive_matrix(self):
+        print plt.matplotlib.__version__
```

Comment:
We are compatible with Python3, so this syntax won't work. In other files, we use 
```
from __future__ import print_function
```
in the head of the file, so we can use `print()`.